extensions/github/src/search-repositories.tsx

```diff
@@ -45,6 +45,14 @@ function SearchRepositories() {
     { keepPreviousData: true },
   );
 
+  useEffect(() => {
+    if (isLoading) {
+      return;
+    }
+
+    history.forEach((repository) => data?.find((r) => r.id === repository.id && visitRepository(r)));
+  }, [data, isLoading]);
```

Comment:
How is `isLoading` useful to the hook?

```suggestion
  useEffect(() => {
    history.forEach((repository) => data?.find((r) => r.id === repository.id && visitRepository(r)));
  }, [data]);
```